src/reanimated2/animation/spring.ts

```diff
@@ -140,6 +140,7 @@ export function withSpring(
       animation: SpringAnimation
     ) {
       return (
+        previousAnimation?.lastTimestamp &&
```

Comment:
This is needed to make `triggered twice` work good, once I've removed `value = 0` in utils.